.spelling

```diff
@@ -772,6 +773,7 @@ inband
 index.html
 index.md
 Infiniband
+init
```

Comment:
```suggestion
```

I wouldn't think we would want this in the dictionary as a word. The only time I see it used outside of backticks is in the phrase "Prepare Site Init". Probably that should be changed to either: 
```text
Prepare `site-init`
```

or maybe

```text
Prepare Site Initialization Files
```